packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -557,7 +559,7 @@ class GutterItem extends StatelessWidget {
     final theme = Theme.of(context);
 
     final foregroundColor = theme.isDarkTheme
-        ? theme.textTheme.bodyText2.color
+        ? theme.textTheme.bodyText2!.color
```

Comment:
handle this being null gracefully.  This looks like a sign we are using themes a bit wrong.

Reply:
@kenzieschmoll any thoughts on what color we should be using for cases like this?

Reply:
nm. related comment seems resolved.

Reply:
I think there are cases where it is okay to null-assert. For example, if we have this value defined in our theme, it will never be null in our app, even though it is nullable in the framework. So anywhere where we were previously assuming a theme value was non-null, it is safe to continue to assume that.